From e06874cc50044932fdc526c55b9b23bd4d929643 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sun, 1 Oct 2023 22:07:32 -0500
Subject: [PATCH] Replace /etc/resolv.conf with systemd-resolved symlink

If /etc/resolv.conf exists and is not a symlink to the systemd resolver then replace it with one. This replacement can only run during boot.
---
 tmpfiles.d/systemd-resolve.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tmpfiles.d/systemd-resolve.conf b/tmpfiles.d/systemd-resolve.conf
index cb1c56d6a6..f210d59279 100644
--- a/tmpfiles.d/systemd-resolve.conf
+++ b/tmpfiles.d/systemd-resolve.conf
@@ -7,4 +7,4 @@
 
 # See tmpfiles.d(5) for details
 
-L! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
+L+! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
-- 
2.42.0

